docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
This change doesn't break the linter as far as I can tell. It just removes the backticks from parsed content. For example, here's `apis.IncomingMessage.instanceProperties.statusCode`:

```js
{ 
  name: 'statusCode',
  description: 'An Integer indicating the HTTP response status code.',
  type: 'Integer',
  collection: false
}
```

My two cents: The backticks are great for Electron-specific names like `MenuItem` and `WebRequest`, but for familiar things like `Integer` and `String` it seems more distracting that helpful.